yelp_package/dockerfiles/itest/mesos/Dockerfile

```diff
@@ -15,4 +15,4 @@
 FROM ubuntu:trusty
 RUN echo "deb http://repos.mesosphere.com/ubuntu trusty main" > /etc/apt/sources.list.d/mesosphere.list
 RUN apt-key adv --keyserver keyserver.ubuntu.com --recv 81026D0004C44CF7EF55ADF8DF7D54CBE56151BF
-RUN apt-get update && apt-get -y install mesos=0.23.1-0.2.61.ubuntu1404
+RUN apt-get update && apt-get -y install mesos=0.24.1-0.2.35.ubuntu1404
```

Comment:
Can you actually undo *these* changes (the upgraded mesos parts) first? 

To answer the question of: will this bump of mesos-cli work on our existing infra?